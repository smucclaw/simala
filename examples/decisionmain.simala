-- run this for example via:
--
-- cabal run -- simala examples/decisiondecl.simala examples/decisioninput.simala examples/decisionmain.simala
--
#eval get_opinion(business, investor)
