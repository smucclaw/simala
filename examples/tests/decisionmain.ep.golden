#eval get_opinion(business, investor)
